cmd/nerdctl/container/container_prune.go

```diff
@@ -58,29 +62,29 @@ func containerPruneAction(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
-	client, ctx, cancel, err := newClient(cmd)
+	client, ctx, cancel, err := nerdClient.NewClient(cmd)
 	if err != nil {
 		return err
 	}
 	defer cancel()
 
-	return containerPrune(ctx, cmd, client)
+	return Prune(ctx, cmd, client)
 }
 
-func containerPrune(ctx context.Context, cmd *cobra.Command, client *containerd.Client) error {
+func Prune(ctx context.Context, cmd *cobra.Command, client *containerd.Client) error {
```

Comment:
If we only use `cmd.OutOrStdout` within the `Prune`, maybe we can change `cmd *cobra.Command` to `writer io.Writer` and then pass in `cmd.OutOrStdout`.

This way we can abstract the func to `Prune containers and write output to the writer` and decouple from `cmd`. (probably we can also move it to the `pkg/containerutil`)